05-13 15:18:58 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" already exists; SQL statement:
CREATE TABLE TEST( ID INT PRIMARY KEY, NAME VARCHAR(25)) [42101-171]
05-14 09:27:05 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE INSTALLATION(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN[*] KEY"; expected "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE INSTALLATION(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN KEY [42001-171]
05-14 09:34:41 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE TEST(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN[*] KEY REFERENCES SERVER(ID), DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID))"; expected "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE TEST(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN KEY REFERENCES SERVER(ID), DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)) [42001-171]
05-14 09:35:29 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE INSTALLATION(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN[*] KEY REFERENCES SERVER(ID), DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)) "; expected "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE iNSTALLATION(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN KEY REFERENCES SERVER(ID), DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)) [42001-171]
05-14 10:01:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE INSTALLATION(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN[*] KEY REFERENCES SERVER(ID), DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID))"; expected "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE INSTALLATION(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN KEY REFERENCES SERVER(ID), DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)) [42001-171]
05-14 10:01:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE INSTALLATION(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN[*] KEY REFERENCES SERVER(ID), DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID))"; expected "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE INSTALLATION(ID INT PRIMARY KEY, SERVER_IDFS INT FOREIGN KEY REFERENCES SERVER(ID), DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)) [42001-171]
05-14 10:03:15 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE INSTALLATION
(ID INT PRIMARY KEY, 
SERVER_IDFS INT FOREIGN[*] KEY REFERENCES SERVER(ID), 
DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), 
VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), 
STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)
)"; expected "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE INSTALLATION
(ID INT PRIMARY KEY, 
SERVER_IDFS INT FOREIGN KEY REFERENCES SERVER(ID), 
DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), 
VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), 
STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)
) [42001-171]
05-14 10:04:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE INSTALLATION
(ID INT PRIMARY KEY, 
SERVER_IDFS INT FOREIGN[*] KEY REFERENCES SERVER(ID), 
DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), 
VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), 
STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)
)"; expected "(, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE INSTALLATION
(ID INT PRIMARY KEY, 
SERVER_IDFS INT FOREIGN KEY REFERENCES SERVER(ID), 
DEPLOYARTEFAKT_IDFS INT FOREIGN KEY REFERENCES DEPLOYARTEFAKT(ID), 
VERSION_IDFS INT FOREIGN KEY REFERENCES VERSION(ID), 
STAGE_IDFS INT FOREIGN KEY REFERENCES STAGE(ID)
) [42001-171]
05-14 10:10:10 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1391)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:930)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:904)
	at org.h2.jdbc.JdbcStatement.getUpdateCount(JdbcStatement.java:221)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1319)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
05-14 10:13:31 jdbc[2]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:645)
	at org.h2.server.web.WebApp.login(WebApp.java:900)
	at org.h2.server.web.WebApp.process(WebApp.java:205)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 14 more
05-14 13:46:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO STAGE VALUES(3, 'Production'):[*] "; SQL statement:
insert into stage Values(3, 'Production'): [42000-171]
05-14 13:46:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO STAGE VALUES(3, 'Production'):[*] "; SQL statement:
insert into stage Values(3, 'Production'): [42000-171]
05-14 14:02:17 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER TABLE DEPLOYARTEFAKT 
ADD GROUPID VARCHAR(255),[*] ARTEFAKTID VARCHAR(255), EXTENTION VARCHAR(255) "; SQL statement:
Alter table DEPLOYARTEFAKT 
add GROUPID VARCHAR(255), ARTEFAKTID VARCHAR(255), EXTENTION VARCHAR(255) [42000-171]
05-14 14:02:37 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ADD[*] ARTEFAKTID VARCHAR(255)"; expected "ALTER, ANALYZE, {"; SQL statement:
ADD ARTEFAKTID VARCHAR(255) [42001-171]
05-14 14:02:37 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ADD[*] EXTENTION VARCHAR(255)"; expected "ALTER, ANALYZE, {"; SQL statement:
ADD EXTENTION VARCHAR(255) [42001-171]
05-14 14:02:49 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER TABLE DEPLOYARTEFAKT 
ADD ARTEFAKTID VARCHAR(255),[*]
ADD EXTENTION VARCHAR(255) "; SQL statement:
Alter table DEPLOYARTEFAKT 
ADD ARTEFAKTID VARCHAR(255),
ADD EXTENTION VARCHAR(255) [42000-171]
05-15 09:53:11 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "STAGE0_.STAGE_ID" not found; SQL statement:
select stage0_.Stage_ID as Stage1_29_, stage0_.Name as Name29_ from Stage stage0_ [42122-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.expression.Alias.optimize(Alias.java:47)
	at org.h2.command.dml.Select.prepare(Select.java:803)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:629)
	at org.jboss.jca.adapters.jdbc.BaseWrapperManagedConnection.doPrepareStatement(BaseWrapperManagedConnection.java:677)
	at org.jboss.jca.adapters.jdbc.BaseWrapperManagedConnection.prepareStatement(BaseWrapperManagedConnection.java:663)
	at org.jboss.jca.adapters.jdbc.WrappedConnection.prepareStatement(WrappedConnection.java:404)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.hibernate.engine.jdbc.internal.proxy.ConnectionProxyHandler.continueInvocation(ConnectionProxyHandler.java:138)
	at org.hibernate.engine.jdbc.internal.proxy.AbstractProxyHandler.invoke(AbstractProxyHandler.java:81)
	at com.sun.proxy.$Proxy45.prepareStatement(Unknown Source)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:147)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:166)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareQueryStatement(StatementPreparerImpl.java:145)
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:1711)
	at org.hibernate.loader.Loader.doQuery(Loader.java:828)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:289)
	at org.hibernate.loader.Loader.doList(Loader.java:2438)
	at org.hibernate.loader.Loader.doList(Loader.java:2424)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2254)
	at org.hibernate.loader.Loader.list(Loader.java:2249)
	at org.hibernate.loader.hql.QueryLoader.list(QueryLoader.java:470)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.list(QueryTranslatorImpl.java:355)
	at org.hibernate.engine.query.spi.HQLQueryPlan.performList(HQLQueryPlan.java:195)
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1248)
	at org.hibernate.internal.QueryImpl.list(QueryImpl.java:101)
	at org.hibernate.ejb.QueryImpl.getResultList(QueryImpl.java:256)
	at ch.lepeit.brain2.BrainService.getAllStages(BrainService.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.jboss.as.ee.component.ManagedReferenceMethodInterceptorFactory$ManagedReferenceMethodInterceptor.processInvocation(ManagedReferenceMethodInterceptorFactory.java:72)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.invocation.InterceptorContext$Invocation.proceed(InterceptorContext.java:374)
	at org.jboss.as.weld.ejb.Jsr299BindingsInterceptor.doMethodInterception(Jsr299BindingsInterceptor.java:129)
	at org.jboss.as.weld.ejb.Jsr299BindingsInterceptor.processInvocation(Jsr299BindingsInterceptor.java:137)
	at org.jboss.as.ee.component.interceptors.UserInterceptorFactory$1.processInvocation(UserInterceptorFactory.java:36)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.invocation.WeavedInterceptor.processInvocation(WeavedInterceptor.java:53)
	at org.jboss.as.ee.component.interceptors.UserInterceptorFactory$1.processInvocation(UserInterceptorFactory.java:36)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ejb3.component.invocationmetrics.ExecutionTimeInterceptor.processInvocation(ExecutionTimeInterceptor.java:43)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.jpa.interceptor.SBInvocationInterceptor.processInvocation(SBInvocationInterceptor.java:47)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.weld.ejb.EjbRequestScopeActivationInterceptor.processInvocation(EjbRequestScopeActivationInterceptor.java:74)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.invocation.InitialInterceptor.processInvocation(InitialInterceptor.java:21)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.invocation.ChainedInterceptor.processInvocation(ChainedInterceptor.java:61)
	at org.jboss.as.ee.component.interceptors.ComponentDispatcherInterceptor.processInvocation(ComponentDispatcherInterceptor.java:53)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ejb3.component.pool.PooledInstanceInterceptor.processInvocation(PooledInstanceInterceptor.java:51)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ejb3.tx.CMTTxInterceptor.invokeInOurTx(CMTTxInterceptor.java:227)
	at org.jboss.as.ejb3.tx.CMTTxInterceptor.required(CMTTxInterceptor.java:303)
	at org.jboss.as.ejb3.tx.CMTTxInterceptor.processInvocation(CMTTxInterceptor.java:189)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ejb3.component.interceptors.CurrentInvocationContextInterceptor.processInvocation(CurrentInvocationContextInterceptor.java:41)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ejb3.component.interceptors.ShutDownInterceptorFactory$1.processInvocation(ShutDownInterceptorFactory.java:42)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ejb3.component.interceptors.LoggingInterceptor.processInvocation(LoggingInterceptor.java:59)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ee.component.NamespaceContextInterceptor.processInvocation(NamespaceContextInterceptor.java:50)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ejb3.component.interceptors.AdditionalSetupInterceptor.processInvocation(AdditionalSetupInterceptor.java:32)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.as.ee.component.TCCLInterceptor.processInvocation(TCCLInterceptor.java:45)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.invocation.ChainedInterceptor.processInvocation(ChainedInterceptor.java:61)
	at org.jboss.as.ee.component.ViewService$View.invoke(ViewService.java:165)
	at org.jboss.as.ee.component.ViewDescription$1.processInvocation(ViewDescription.java:176)
	at org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:288)
	at org.jboss.invocation.ChainedInterceptor.processInvocation(ChainedInterceptor.java:61)
	at org.jboss.as.ee.component.ProxyInvocationHandler.invoke(ProxyInvocationHandler.java:72)
	at ch.lepeit.brain2.BrainService$$$view4.getAllStages(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.jboss.weld.util.reflection.SecureReflections$13.work(SecureReflections.java:267)
	at org.jboss.weld.util.reflection.SecureReflectionAccess.run(SecureReflectionAccess.java:52)
	at org.jboss.weld.util.reflection.SecureReflectionAccess.runAsInvocation(SecureReflectionAccess.java:137)
	at org.jboss.weld.util.reflection.SecureReflections.invoke(SecureReflections.java:263)
	at org.jboss.weld.bean.proxy.EnterpriseBeanProxyMethodHandler.invoke(EnterpriseBeanProxyMethodHandler.java:110)
	at org.jboss.weld.bean.proxy.EnterpriseTargetBeanInstance.invoke(EnterpriseTargetBeanInstance.java:56)
	at org.jboss.weld.bean.proxy.ProxyMethodHandler.invoke(ProxyMethodHandler.java:105)
	at ch.lepeit.brain2.BrainService$Proxy$_$$_Weld$Proxy$.getAllStages(BrainService$Proxy$_$$_Weld$Proxy$.java)
	at ch.lepeit.brain2.BrainRESTService.listAllStages(BrainRESTService.java:30)
	at ch.lepeit.brain2.BrainRESTService$Proxy$_$$_WeldClientProxy.listAllStages(BrainRESTService$Proxy$_$$_WeldClientProxy.java)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
	at org.jboss.resteasy.core.ResourceMethod.invokeOnTarget(ResourceMethod.java:257)
	at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:222)
	at org.jboss.resteasy.core.ResourceMethod.invoke(ResourceMethod.java:211)
	at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:525)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:502)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:119)
	at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
	at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
	at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:847)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:329)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:248)
	at org.jboss.weld.servlet.ConversationPropagationFilter.doFilter(ConversationPropagationFilter.java:62)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:280)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:248)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:275)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:161)
	at org.jboss.as.jpa.interceptor.WebNonTxEmCloserValve.invoke(WebNonTxEmCloserValve.java:50)
	at org.jboss.as.web.security.SecurityContextAssociationValve.invoke(SecurityContextAssociationValve.java:153)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:155)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:368)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:877)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:679)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:931)
	at java.lang.Thread.run(Thread.java:722)
05-21 09:59:38 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM DEPLOYARTEFAKT INSTALLATION SERVER[*] STAGE"; SQL statement:
SELECT * FROM DEPLOYARTEFAKT INSTALLATION SERVER STAGE [42000-171]
05-22 14:58:38 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1391)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1376)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:462)
	at org.jboss.jca.adapters.jdbc.BaseWrapperManagedConnection.destroy(BaseWrapperManagedConnection.java:436)
	at org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreArrayListManagedConnectionPool.doDestroy(SemaphoreArrayListManagedConnectionPool.java:798)
	at org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreArrayListManagedConnectionPool.flush(SemaphoreArrayListManagedConnectionPool.java:561)
	at org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreArrayListManagedConnectionPool.shutdown(SemaphoreArrayListManagedConnectionPool.java:667)
	at org.jboss.jca.core.connectionmanager.pool.AbstractPool.shutdown(AbstractPool.java:595)
	at org.jboss.jca.core.connectionmanager.AbstractConnectionManager.shutdown(AbstractConnectionManager.java:158)
	at org.jboss.as.connector.subsystems.datasources.AbstractDataSourceService.stop(AbstractDataSourceService.java:140)
	at org.jboss.msc.service.ServiceControllerImpl$StopTask.stopService(ServiceControllerImpl.java:1911)
	at org.jboss.msc.service.ServiceControllerImpl$StopTask.run(ServiceControllerImpl.java:1874)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:722)
05-22 15:07:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Unknown data type: "("; SQL statement:
alter table stage 
add varchar(3) kurzname [50004-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:3661)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3524)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:5000)
	at org.h2.command.Parser.parseAlterTable(Parser.java:4839)
	at org.h2.command.Parser.parseAlter(Parser.java:4322)
	at org.h2.command.Parser.parsePrepared(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.parse(Parser.java:250)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
05-28 09:00:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO STAGE 
WHERE[*] ID = 1 
KURZNAME = DEV "; expected "., (, DIRECT, SORTED, DEFAULT, VALUES, SET, (, SELECT, FROM"; SQL statement:
insert into stage 
where id = 1 
Kurzname = DEV [42001-171]
05-28 09:04:17 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO STAGE ( KURZNAME )
WHERE[*] ID = 1
VALUES ('DEV')"; expected "DIRECT, SORTED, DEFAULT, VALUES, SET, (, SELECT, FROM"; SQL statement:
INSERT INTO STAGE ( KURZNAME )
WHERE ID = 1
VALUES ('DEV') [42001-171]
05-28 09:06:34 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Column "DEV" not found; SQL statement:
UPDATE STAGE 
SET KURZNAME = DEV
WHERE ID = 1 [42122-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Update.prepare(Update.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:13:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1391)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1376)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:462)
	at org.jboss.jca.adapters.jdbc.BaseWrapperManagedConnection.destroy(BaseWrapperManagedConnection.java:436)
	at org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreArrayListManagedConnectionPool.doDestroy(SemaphoreArrayListManagedConnectionPool.java:798)
	at org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreArrayListManagedConnectionPool.flush(SemaphoreArrayListManagedConnectionPool.java:561)
	at org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreArrayListManagedConnectionPool.shutdown(SemaphoreArrayListManagedConnectionPool.java:667)
	at org.jboss.jca.core.connectionmanager.pool.AbstractPool.shutdown(AbstractPool.java:595)
	at org.jboss.jca.core.connectionmanager.AbstractConnectionManager.shutdown(AbstractConnectionManager.java:158)
	at org.jboss.as.connector.subsystems.datasources.AbstractDataSourceService.stop(AbstractDataSourceService.java:140)
	at org.jboss.msc.service.ServiceControllerImpl$StopTask.stopService(ServiceControllerImpl.java:1911)
	at org.jboss.msc.service.ServiceControllerImpl$StopTask.run(ServiceControllerImpl.java:1874)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:15:21 jdbc[2]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:645)
	at org.h2.server.web.WebApp.login(WebApp.java:900)
	at org.h2.server.web.WebApp.process(WebApp.java:205)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:645)
	at org.h2.server.web.WebApp.login(WebApp.java:900)
	at org.h2.server.web.WebApp.process(WebApp.java:205)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 14 more
06-05 09:16:11 jdbc[3]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:645)
	at org.h2.server.web.WebApp.login(WebApp.java:900)
	at org.h2.server.web.WebApp.process(WebApp.java:205)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:645)
	at org.h2.server.web.WebApp.login(WebApp.java:900)
	at org.h2.server.web.WebApp.process(WebApp.java:205)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 14 more
06-05 09:22:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO DEPLOYARTEFAKT 
VALUE[*] (2,""csi-ear"",""com.axa.ch.csi"",""csi-ear"",""ear"" "; expected "., (, DIRECT, SORTED, DEFAULT, VALUES, SET, (, SELECT, FROM"; SQL statement:
Insert into DEPLOYARTEFAKT 
Value (2,"csi-ear","com.axa.ch.csi","csi-ear","ear" [42001-171]
06-05 09:23:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO DEPLOYARTEFAKT 
VALUE[*] (2,""csi-ear"",""com.axa.ch.csi"",""csi-ear"",""ear"") "; expected "., (, DIRECT, SORTED, DEFAULT, VALUES, SET, (, SELECT, FROM"; SQL statement:
Insert into DEPLOYARTEFAKT 
Value (2,"csi-ear","com.axa.ch.csi","csi-ear","ear") [42001-171]
06-05 09:24:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "csi-ear" not found; SQL statement:
Insert into DEPLOYARTEFAKT 
Values (2,"csi-ear","com.axa.ch.csi","csi-ear","ear") [42122-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:34:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet','0','server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:35:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet','0','server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:35:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet','0','server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:36:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet',0,'server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:36:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet',0,'server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:36:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet',0,'server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:36:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet',0,'server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:38:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT FROM DEPLOYARTEFAKT"; expected "., (, AS, RIGHT, LEFT, FULL, INNER, JOIN, CROSS, NATURAL, ,, SELECT"; SQL statement:
SELECT FROM DEPLOYARTEFAKT [42001-171]
06-05 09:38:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet',0,'server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
06-05 09:39:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO DEPLOYARTEFAKT 
VALUES (1,'lxfapp26.undefined.firma.intranet',0,'server 1 (oms)') [21002-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:417)
	at org.h2.engine.Session.prepareCommand(Session.java:366)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1312)
	at org.h2.server.web.WebApp.query(WebApp.java:1002)
	at org.h2.server.web.WebApp$1.next(WebApp.java:965)
	at org.h2.server.web.WebApp$1.next(WebApp.java:954)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:722)
